### PR TITLE
Add healthcheck commands to dockerfiles

### LINKED ISSUE #63: Docker Healthcheck command
Hello,

Is it possible to add a healthcheck to the dockerfiles?

It should be as simple as adding the following line to the end of each dockerfile

`HEALTHCHECK CMD curl --fail http://localhost:8001 || exit 1`

I have not tested this myself (nor have i actually made the patch) as I'm not at my workstation right now, and obviously this only tests if the web interface is running.

### PR DESCRIPTION
**Pull request by andrew-stclair**
Resolves #63 

i have tested `Dockerfile`, but have not tested `Dockerfile.raspberry` or `Dockerfile.raspberry-k8s`

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -16,3 +16,4 @@ EXPOSE 8001
 STOPSIGNAL SIGINT
 
 ENTRYPOINT [ "python", "start.py" ]
+HEALTHCHECK --interval=5s --timeout=3s CMD /home/warrior/data/wget-at -nv -t1 'http://localhost:8001/index.html' --delete-after || exit 1
```
Comment (Eeems) [inline Dockerfile:19]:
```suggestion
HEALTHCHECK --interval=5s --timeout=3s CMD /home/warrior/data/wget-at -nv -t1 'http://localhost:8001/index.html' -O - 1> /dev/null || exit 1
```
This avoids downloading and deleting the file and instead just pipes the contents to `/dev/null`

Comment (Fusl) [reply Dockerfile]: `-O /dev/null` to avoid having to write to stdout and redirect stdout to /dev/null ;)

Comment (Fusl) [reply Dockerfile]: also, mind updating the other dockerfiles as well (even though i think they're not used)?

Comment (Arkiver2) [reply Dockerfile]: When suggested changes are in by @andrew-stclair, this PR will be merged.

Comment (andrew-stclair) [reply Dockerfile]: Suggestions have been implemented

**Pull request merged**